menuItems.h

```diff
@@ -22,6 +22,7 @@
 #define _MENUTEMS_
 
 #include <windows.h>
+#include "playerData.h"
```

Comment:
This include should not be done
menuItems.h **only** modifies the values in selection[] array. It should not call any external functions. Except showHelp() and showAbout(), all other functions are called via main.cpp